ngrinder-controller/src/main/resources/ngrinder_datachange_logfile/db.changelog_schema_28.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+				   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+				   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-2.0.xsd">
+	<changeSet author="ngrinder.3.5.0" id="28" dbms="h2">
+		<comment>Add default value to timestamp data type.</comment>
+		<modifyDataType tableName="NUSER" columnName="created_date" newDataType="TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP"/>
+		<modifyDataType tableName="NUSER" columnName="last_modified_date" newDataType="TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP"/>
+
+		<modifyDataType tableName="PERF_TEST" columnName="created_date" newDataType="TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP"/>
+		<modifyDataType tableName="PERF_TEST" columnName="last_modified_date" newDataType="TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP"/>
+		<modifyDataType tableName="PERF_TEST" columnName="finish_time" newDataType="TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP"/>
```

Comment:
Only apply this on created and last_modied not finish/start/scheduled...

Reply:
Yes, I'll modify mysql too.

Reply:
Okay

Reply:
but https://github.com/naver/ngrinder/pull/368 said if default value doesn't assign to `timestamp` data type specific version of mysql occur error..

It looks like need to check.

Reply:
If true(error occured in specific version of mysql), 

Even if `finish/start/scheduled` has a default value, it doesn't seem to be a problem since these values ​​are always explicitly assigned.